ios/FluentUI/Navigation/SideBar.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Delegate protocol to handle user events inside the side bar.
+@objc(MSFSideBarDelegate)
+public protocol SideBarDelegate {
+    /// Called after the view representing `TabBarItem` is selected.
+    @objc optional func sideBar(_ sideBar: SideBar, didSelect item: TabBarItem, fromTop: Bool)
+
+    /// Called after the avatar view is tapped in the side bar.
+    @objc optional func sideBar(_ sideBar: SideBar, didActivate avatarView: AvatarView)
+}
+
+/// View for a vertical side bar that can be used for app navigation.
+/// Optimized for horizontal regular + vertical regular size class configuration. Prefer using TabBarView for other size class configurations.
+@objc(MSFSideBar)
+open class SideBar: UIView {
```

Comment:
>SideBar [](start = 11, length = 7)

If this is not part of the Fluent design spec (figma, I can send link privately), could you please ask the designer to include it there?

Reply:
Kaan did mention something about adding this to the mobile toolkit and he's working on it.
It is in the figma designs:
https://www.figma.com/file/0tO5Oliq4eyd48dL8gFAj1/WXP-iPad-L1-Fluent-Refresh?node-id=259%3A0